Lib/multiprocessing/reduction.py

```diff
@@ -51,14 +51,35 @@ def dumps(cls, obj, protocol=None):
         cls(buf, protocol).dump(obj)
         return buf.getbuffer()
 
-    loads = pickle.loads
+    @classmethod
+    def loads(cls, bytes_object, *, fix_imports=True,
```

Comment:
Uh... I hadn't noticed these were class methods...

Reply:
The problem is that loads do not need to instantiate a Pickler class so it was designed here as a class method.

Would you prefer it to be a regular method that does the same (defers the call to pickle.loads)?